src/metabase/query_processor/pivot.clj

```diff
@@ -1,35 +1,49 @@
 (ns metabase.query-processor.pivot
-  "Pivot table actions for the query processor"
+  "Pivot table query processor. Runs a bunch of subqueries separately and stitches results together. TODO -- this whole
+  thing is dumb, why don't we just generate a big ol' UNION query? -- Cam"
```

Comment:
I'm struggling to follow the below since I don't understand how we implement pivot queries.

A paragraph or two about how this works would go a long way to help out future readers.